src/test/java/com/epam/tc/hw7/util/JdiDataProvider.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.tc.hw7.util;
+
+import com.epam.tc.hw7.entities.MetalAndColorEntity;
+import com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+import com.google.gson.stream.JsonReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.Map;
+import org.testng.annotations.DataProvider;
+
+public class JdiDataProvider {
+    public static final String EXPECTED_USERNAME = "ROMAN IOVLEV";
```

Comment:
What is the reason of separating "full name" and name&password from User entity??

Reply:
no particular reason, it just seems logical to me. 
maybe should be combined with a user class